dashboard/app/views/levels/_admin.html.haml

```diff
@@ -36,7 +36,7 @@
         %li (Cannot edit)
 
     %br/
-    This level is in #{@level.script_levels.count} scripts:
+    This level is in #{@level.script_levels.to_a.count} scripts:
```

Comment:
Aside, I don't think this prevents a DB dip. The dip isn't caused by `count` but rather by `script_levels`, according to `dashboard-console` locally.